README.md

```diff
@@ -35,3 +35,95 @@ console.log(binaryPacket.toString('hex'))
 
 const jsonPacket = packet.deserializeIlpPayment(binaryPacket)
 ```
+
+### Serialize/deserialize ILQP Quote Requests/Responses
+
+#### QuoteLiquidityRequest
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityRequest({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  destinationAccount: 'example.nexus.bob',
+  destinationHoldDuration: 3000
+})
+
+const json = packet.deserializeIlqpLiquidityRequest(binary)
+```
+
+#### QuoteLiquidityResponse
+
+```js
+const packet = require('ilp-packet')
+
+const binary = packet.serializeIlqpLiquidityResponse({
+  requestId: 'c217449a-69e9-4a08-9575-6e7eafa818c0',
+  liquidityCurve: Buffer.alloc(16), // Must be a buffer of size (n * 16) bytes
```

Comment:
> How about a client that requests a liquidity curve for a sending UI to avoid sending a quote request to the connector every time the user changes the amount?

I can query the curve using the public methods on LiquidityCurve.

Note that the math isn't totally trivial, so I already need to access the curve via some module. I'm not going to be looking at it myself.

So to pass curves between different modules (such as this one and `ilp-routing`) we need a standard format that all the different libraries can support. Why not use a Buffer containing the same encoding format that is already standardized?

* No expensive conversion to strings or bignums
* No dependency on a bignum library
* I don't have to convert it back to a buffer if I want to use an UInt32Array (which is much faster to do curve operations on than an array of bignums)